lms/djangoapps/discussion/templates/discussion/edx_ace/response_notification/email/body.html

```diff
@@ -56,3 +56,7 @@
     </tr>
 </table>
 {% endblock %}
+
+{% block google_analytics_pixel %}
+<img src="{{ ga_pixel_url }}" alt="" role="presentation" aria-hidden="true" />
```

Comment:
`ga_tracking_pixel_url` (or change the key in the context dict to `ga_pixel_url`)

Reply:
😫